components/brave_wallet/browser/eth_tx_manager.cc

```diff
@@ -1183,6 +1194,37 @@ void EthTxManager::OnGetGasEstimation1559(
   std::move(callback).Run(std::move(estimation));
 }
 
+void EthTxManager::OnGetBaseFeePerGas(GetGasEstimation1559Callback callback,
+                                      const std::string& chain_id,
```

Comment:
nit: we don't reference this chain_id, so it might be removed

Reply:
https://github.com/brave/brave-core/pull/18707/commits/9282c58d14c647ce66c9637a5ac665f4f3a3bf6e